jsx/StaticDataTable.js

```diff
@@ -297,6 +297,19 @@ var StaticDataTable = React.createClass({
       return (searchString.indexOf(searchKey) > -1);
     }
 
+    // Handle array inputs for multiselects
+    if (typeof filterData === 'object') {
+      var match = false;
+      for (let i = 0; i < filterData.length; i += 1) {
+        var searchKey2 = filterData[i].toLowerCase();
```

Comment:
definitely not a big deal but just curious as to why you named this `searchKey2` instead of just `searchKey`

Reply:
it was failing Travis since the var searchKey had already been defined before..I could create a more global variable called 'searchKey' instead

Reply:
Or just change the `var` for `let`